tools.go

```diff
@@ -0,0 +1,5 @@
+// +build tools
```

Comment:
Do we want to leave this around _in case_ we need it later?

Reply:
I'm not actually sure how we would use this. Do you know?